draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1626,12 +1633,17 @@ client uses `get-all-by-hash`, then it will already have the new STH.
 
 ### Evaluating compliance
 
-To be considered compliant, a certificate MUST be accompanied by at least one
-valid SCT. A certificate not accompanied by any valid SCTs MUST NOT be
-considered compliant by TLS clients.
+To be considered compliant with this specification, a certificate MUST be
+accompanied by evidence that it has been included in one or more logs. There
+are two types of proof that a client might require:
 
-A TLS client MUST NOT evaluate compliance if it did not send both the
-`transparency_info` and `status_request` TLS extensions in the ClientHello.
+1. A log inclusion proof to a Signed Tree Head
+2. A Signed Certificate Timestamp
+
+It is up to a client's local policy which forms of evidence are sufficient for
```

Comment:
Since you're also proposing that we remove the "Handling of Non-compliance" section, is it also worth explicitly stating here that it's up to the client to decide how to handle non-compliance?

Maybe reword something like this...
"It is entirely up to a client's local policy to specify:
  - which forms and quantity of evidence are considered sufficient to achieve compliance.
  - how to handle non-compliance."